kalite/distributed/static/css/distributed/bootstrap-overrides.less

```diff
@@ -410,6 +410,14 @@ form .input-group {
   text-align: center;
 }
 
+.content-explore-topic .row div.col-xs-5{
```

Comment:
This is not the right fix, as it is overwriting the grid styling.

Better to change the classes on the elements in question to `col-xs-12` - see here for more info: http://getbootstrap.com/css/#grid